### PR TITLE
Enhance User Interface

### PR DESCRIPTION
**Pull request by edenlisk**
In this PR we did the following:
1. enhance user interface
2. Add contact

### CODE REVIEW

**Review by levy002 — COMMENTED** [review]
Hello @edenlisk & @haliljon 
While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :red_circle: :red_circle:  :red_circle: 

You have submitted a project, that doesn't run. Kindly make sure to add the `package.json` file for dependencies so I can be able to run your project locally.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -7,32 +7,69 @@
...
+      </div>
+      <div class="navlist">
+        <ul>
+          <li class="list-items"><a href="" id="list"> List</a></li>
+          <li>|</li>
+          <li class="list-items"><a href="" id="addNew">Add new</a></li>
+          <li>|</li>
+          <li class="list-items"><a href="" id="contact">Contact</a></li>
+        </ul>
+      </div>
```
Comment (OLIPLICHE) [inline index.html:30]: - [ ]  **Please make sure when any of the links(List, Add, Contact) are clicked, the main area changes to display only one of the three content sections.**

Capture
- Kindly use this wireframe as a guide:
Capture
- **Suggestion:** You can add and remove CSS classes to display or hide the content by using the 👉  [Element.classList](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#examples) `add` and `remove` method.

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linter is green (the last commit have a green check mark (✅)) 👏
- Commit messages are meaningful 👏 👏
- GitHub low is correctly used (Pull Request is made from the feature branch into the main branch) 👏

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**You can also consider:**
- **Not to disable the Linter in script.js file**: A linter is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs.
- To Add `Licence` section in your ReadMe, and make the file looks more professional.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TadesseDev — APPROVED** [review]
### Hi @edenlisk 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
68747470733a2f2f6669636f666f72756d732e6d796669636f2e636f6d2f74352f696d6167652f736572766572706167652f696d6167652d69642f343932363069463839303835363833444139334135422f696d6167652d73697

## Highlights
 - [ ] Good job making the changes requested by the previous code reviewer. 🙏
 - [ ]  Correct GitHub Flow 👍🏾
 - [ ]  Good commit history 🥇
 - [ ]  Good PR title/description 💯
 - [ ]  All linters passed 🥇
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**